deploy/api/templates/00-deployment.yml

```diff
@@ -1,36 +1,46 @@
 apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
-  name: "{{ default "search-api" .Values.deploymentName }}"
+  name: "{{ default "search-api" .Values.name }}"
+  namespace: "{{ .Values.namespace }}"
   labels:
     subsystem: "{{ .Values.labels.subsystem }}"
-    container: "{{ default "search-api" .Values.deploymentName }}"
+    container: "{{ default "search-api" .Values.deployment.name }}"
     service-group: api
     log-style: uwsgi
+    env: "{{ .Values.namespace }}"
 spec:
-  replicas: {{ default 1 .Values.replicas }}
+  replicas: {{ default 1 .Values.scaling.replicas }}
   template:
     metadata:
       labels:
         subsystem: "{{ .Values.labels.subsystem }}"
-        container: "{{ default "search-api" .Values.deploymentName }}"
+        container: "{{ default "search-api" .Values.deployment.name }}"
         service-group: api
         log-style: uwsgi
       # annotations:
       #   prometheus.io/scrape: 'true'
     spec:
       containers:
-      - name: "{{ default "search-api" .Values.deploymentName }}"
-        image: arxiv/search-api:{{ .Values.imageTag }}
+      - name: "{{ default "search-api" .Values.deployment.name }}"
+        image: arxiv/search-api:{{ .Values.image.tag }}
         imagePullPolicy: Always
         ports:
         - containerPort: 8000
         env:
+        - name: APPLICATION_ROOT
+          value: "{{ .Values.ingress.path }}"
+        - name: BASE_SERVER
+          value: "{{ .Values.base_server }}"
+        - name: NAMESPACE
```

Comment:
I would expect a namespace env var to be provided to the pods by kubernetes. Does this not happen?